cmd/depth/depth.go

```diff
@@ -19,6 +19,16 @@ const (
 
 var outputJSON bool
 
+type summary struct {
+	numInternal int
+	numExternal int
+	numTesting  int
+}
+
+func newSummary() (sum *summary) {
```

Comment:
This function is probably unnecessary since it uses the zero-values for each field anyways